resources/user-agents/browsers/uc-browser/uc-mini-generic.json

```diff
@@ -27,7 +27,7 @@
           }
         },
         {
-          "match": "UCWEB/2.0 (Linux* Opera Mini/*) U2/* UCMini/*"
```

Comment:
Are there any `UCWEB/2.0 ` patterns left?  If so, they may take priority over these with the asterisk in browscap-php even if they're shorter length (if everything else matches).

Reply:
No. I replaced everyone I found.